Disclaimer - track calls only with Marketing Hub (#3217)
segmentio/segment-docs: src/connections/destinations/catalog/hubspot/index.md

```

> warning "Important"

> > Track calls and custom events in Hubspot require Hubspot Marketing Hub (https://knowledge.hubspot.com/analytics-tools/create-custom-behavioral-events). You must have a HubSpot Enterprise account for Segment to pass traits from an Identify call through to your Track call and send them as[custom events to HubSpot](https://knowledge.hubspot.com/events-user-guide-v2/using-custom-events).

Reviewer: ```suggestion
> Marketing Hub subscription is required to [track calls and custom marketing events](https://knowledge.hubspot.com/analytics-tools/create-custom-behavioral-events) in Hubspot. You must have a HubSpot Enterprise account for Segment to pass traits from an Identify call through to your Track call and send them as [custom events to HubSpot](https://knowledge.hubspot.com/events-user-guide-v2/using-custom-events){:target="_blank"}.
```

> > Track calls and custom events in Hubspot require Hubspot Marketing Hub (https://knowledge.hubspot.com/analytics-tools/create-custom-behavioral-events). You must have a HubSpot Enterprise account for Segment to pass traits from an Identify call through to your Track call and send them as[custom events to HubSpot](https://knowledge.hubspot.com/events-user-guide-v2/using-custom-events).

Reviewer: ```suggestion
> Marketing Hub subscription is required to [track calls and custom marketing events](https://knowledge.hubspot.com/analytics-tools/create-custom-behavioral-events) in Hubspot. You must have a HubSpot Enterprise account for Segment to pass traits from an Identify call to your Track call and send them as [custom events to HubSpot](https://knowledge.hubspot.com/events-user-guide-v2/using-custom-events){:target="_blank"}.

> > Marketing Hub subscription is required to [track calls and custom marketing events](https://knowledge.hubspot.com/analytics-tools/create-custom-behavioral-events) in Hubspot. You must have a HubSpot Enterprise account for Segment to pass traits from an Identify call to your Track call and send them as [custom events to HubSpot](https://knowledge.hubspot.com/events-user-guide-v2/using-custom-events){:target="_blank"}.

Reviewer: ```suggestion
> Marketing Hub subscription is required to [track calls and custom marketing events](https://knowledge.hubspot.com/analytics-tools/create-custom-behavioral-events){:target="_blank"} in HubSpot. You must have a HubSpot Enterprise account for Segment to pass traits from an Identify call to your Track call and send them as [custom events to HubSpot](https://knowledge.hubspot.com/events-user-guide-v2/using-custom-events){:target="_blank"}.
```
